assets/src/domain/eventEditor/ui/tickets/hooks/useDeleteTicketHandler.ts

```diff
@@ -1,36 +1,58 @@
 import { useCallback } from 'react';
 
 import useTicketPrices from '@edtrServices/apollo/queries/tickets/useTicketPrices';
-import { EntityId } from '@dataServices/types';
-import { useTicketMutator, usePriceMutator } from '@edtrServices/apollo/mutations';
+import type { EntityId } from '@dataServices/types';
```

Comment:
oooooo new TS Voodoo...  what does this do?

Reply:
No, not new, you forgot your [own comment](https://github.com/eventespresso/event-espresso-core/pull/2608#discussion_r401781465) 😄

Reply:
well of course I did, that was months ago